File: old_baser/javarecipe/src/main/java/work/shion/javarecipe/pages/entrypoint/views/Tab1stPageFragment.java
package work.shion.javarecipe.pages.entrypoint.views;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import work.shion.javarecipe.R;


/**
 * タブコンテンツその１用のFragment
 */
public class Tab1stPageFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
            @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.entrypoint_fragment_tab_1st, container, false);
    }
}

File: old_baser/ktrecipe/src/main/java/work/shion/ktrecipe/pages/entrypoint/views/Tab1stPageFragment.kt
package work.shion.ktrecipe.pages.entrypoint.views

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import work.shion.ktrecipe.R


/**
 * タブコンテンツその１用のFragment
 */
class Tab1stPageFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.entrypoint_fragment_tab_1st, container, false)
    }
}
